mutable/slice_test.go

```diff
@@ -0,0 +1,94 @@
+package mutable
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFilter(t *testing.T) {
+	t.Parallel()
+	is := assert.New(t)
```

Comment:
why not `table driven` ?

Reply:
what do you mean ?

Reply:
https://go.dev/wiki/TableDrivenTests

Reply:
Ok 👌

I usually don't write table-driven tests for only 2 or 3 test cases.

Reply:
i used to do the same, but i always got reviews about it.. so if it is one case only then ok, otherwise always use tables